config/201-clusterrolebinding.yaml

```diff
@@ -77,3 +77,20 @@ roleRef:
   kind: ClusterRole
   name: knative-eventing-webhook
   apiGroup: rbac.authorization.k8s.io
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This also needs a role for `addressable-resolver`

Reply:
fixed. Thanks.